lib/sharezone_localizations/l10n/app_de.arb

```
@@ -5,9 +5,236 @@
+    "@_CHANGE_EMAIL_ADDRESS_PAGE": {},
+    "changeEmailAddressPageTitle": "E-Mail ändern",
+    "changeEmailAddressPageCurrentEmailTextfieldLabel": "Aktuell",
+    "@changeEmailAddressPageCurrentEmailTextfieldLabel": {
+        "description": "The label for the text field which is used for the current email address"
+    },
+    "changeEmailAddressPageNewEmailTextfieldLabel": "Neu",
+    "@changeEmailAddressPageNewEmailTextfieldLabel": {
+        "description": "The label for the text field which is used for the new email address"
+    },
+    "changeEmailAddressPagePasswordTextfieldLabel": "Passwort",
+    "@changeEmailAddressPageNewEmailTextfieldLabel": {
```

rubocop (1.69.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)